### PR TITLE
Added german translations

### PR DESCRIPTION
**Pull request by SirWindfield**
If anyone has something to complain about the translation, let me know ^_^

### CODE REVIEW

**Comment by Landmaster** [comment]
@SirWindfield There's one more file to translate—[PlusTiC/src/main/resources/assets/tconstruct/lang/en_US.lang](https://github.com/Landmaster/PlusTiC/blob/master/src/main/resources/assets/tconstruct/lang/en_US.lang).

File: src/main/resources/assets/plustic/lang/de_DE.lang [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+item.mirioningot.name=Mirionbarren
+item.invarnugget.name=Invarklumpen
+item.invaringot.name=Invarbarren
+
+tile.osmiridiumblock.name=Osmiridiumblock
+tile.alumiteblock.name=Alumiteblock
+tile.mirionblock.name=Mirionblock
+tile.invarblock.name=Invarblock
+
+msg.plustic.globalmodifier.set=Linked inventory at [x=%d, y=%d, z=%d] and dimension %d
```
Comment (Landmaster) [inline src/main/resources/assets/plustic/lang/de_DE.lang]: As this is the same as tooltip.plustic.globalmodifier.info, the localization could use that of tooltip.plustic.globalmodifier.info, if I'm correct. I'm not completely sure, though.

**Comment by Landmaster** [comment]
@SirWindfield I'll be adding a significant amount of content in the next release (including the laser gun in #30), so this translation will probably remain incomplete for some time (also, see comments above). I'll leave this open for now so that those translations can be completed. But thank you very much for the localizations though!

**Comment by SirWindfield** [comment]
@Landmaster I'll wait then before pushing the new content.

**Comment by Landmaster** [comment]
@SirWindfield Actually, I've already released the content recently in 5.0.0.0, so you can start pushing the new content now: I will mainly do bugfixes for the next release.

**Comment by Landmaster** [comment]
@SirWindfield I'm moving PlusTiC to the 1.12 branch (for the Minecraft 1.12 update); can you open a new pull request to that branch? Thanks.

**Pull request closed without merging**